contrib/mpi-proxy-split/split_process.cpp

```diff
@@ -352,12 +406,14 @@ setLhMemRange()
   close(mapsfd);
   static MemRange_t lh_mem_range;
   if (found) {
-#if !defined(USE_MANA_LH_FIXED_ADDRESS)
-    lh_mem_range.start = (VA)area.addr - TWO_GB;
-    lh_mem_range.end = (VA)area.addr - ONE_GB;
+#if defined(CENTOS)
+    findLHMemRange(&lh_mem_range);
```

Comment:
Why add `CENTOS` back? cori has `USE_MANA_LH_FIXED_ADDRESS` defined by default, doesn't it?

Reply:
yes, but this was the initial implementation of it. once i've done tests on cori, we'll just have this be the default way the mem range is determined.

Reply:
Yes, so on cori, it will go
```
#else
    lh_mem_range.start = 0x2aab00000000;
    lh_mem_range.end =   0x2aab00000000 + ONEGB;
#endif
```
Correct?
What I meant is you don't need to add `CENTOS`, just change the code for when `USE_MANA_LH_FIXED_ADDRESS` is not defined. That name implies everything, which is better. Other people won't understand what makes `CENTOS` different.
One day if we don't need `USE_MANA_LH_FIXED_ADDRESS` anymore, then we can remove that as well.

Reply:
So on cori, it will go
```
#else
    lh_mem_range.start = 0x2aab00000000;
    lh_mem_range.end =   0x2aab00000000 + ONEGB;
#endif
```
Correct?
What I meant is you don't need to add CENTOS, just change the code for when USE_MANA_LH_FIXED_ADDRESS is not defined. That name implies everything, which is better. Other people won't understand what makes CENTOS different.
One day if we don't need USE_MANA_LH_FIXED_ADDRESS anymore, then we can remove that as well.